main.py

```diff
@@ -130,6 +131,13 @@ def executeCommandList():
     for command in linesFromFile(custom_command_file, deleteFetched=True):
         runCommand(command)
 
+def executeCustomShellScript():
+    log('Running custom shell script...')
+    dir = os.path.dirname(__file__)
+    custom_script = os.path.join(dir, 'scripts/custom-command.sh')
+    os.chmod(custom_script, 0o755)
+    os.system(custom_script)
```

Comment:
I was not able to get this to run properly...
I tried different variations in my custom-command file:
```
/Volumes/pi\'s\ home/MCCrafting/update_recipes.sh
/Volumes/pi\'s\ home/MCCrafting/./update_recipes.sh
```

But always received this error:

```
pi@mikey-minepi:~/MIEMCServer $ python main.py -k
...
Running custom shell script...
sh: 1: /home/pi/MIEMCServer/scripts/custom-command.sh: Text file busy
```

Reply:
Yeah, I'm not entirely sure what the issue here is. I've tried this on both the server and on the Ubuntu distro on my Mac, and they both work fine.

Reply:
Can you share how you are testing this? What is in your `custom-command.sh` file?